frontend/src/features/agent/functions/formatDescription.ts

```diff
@@ -26,9 +26,15 @@ export const formatDescription = (tool: AgentTool, t: TFunction) => {
       )}`;
     case 'internet_search':
       return `${t(`agent.tools.internet_search.name`)}:${t(
-        `agent.tools.internet_search.desciription`
+        `agent.tools.internet_search.description`
       )}`;
     default:
+      if (exists(`agent.tools.${tool.name}`)) {
```

Comment:
Could you remove existing definitions e.g. get_weather, sql_db_query etc because they are already declared on i18n files.

Reply:
It would also be appreciated if you could remove following part on [AGENT.md](https://github.com/aws-samples/bedrock-claude-chat/blob/v1/docs/AGENT.md).

>Edit [formatDescription.ts](https://github.com/aws-samples/bedrock-claude-chat/blob/v1/frontend/src/features/agent/functions/formatDescription.ts) so that frontend app can refer to it correctly.

Reply:
I have now done this